frontendActions/src/main/resources/actions/ResourceBundle.properties

```diff
@@ -26,7 +26,7 @@ action.GitMachete.FastForwardParentToMatchSelectedBranchAction.GitMacheteContext
 action.GitMachete.FastForwardParentToMatchSelectedBranchAction.description=Fast forward parent to match selected branch
 
 action.GitMachete.SlideOutSelectedBranchAction.text=Git Machete: Slide Out Selected Branch
-action.GitMachete.SlideOutSelectedBranchAction.GitMacheteContextMenu.text=_Slide Out Branch
+action.GitMachete.SlideOutSelectedBranchAction.GitMacheteContextMenu.text=Slide _Out Branch
```

Comment:
it was colliding with reset

Reply:
Looks like a candidate for an automatic check

Reply:
Check script added but still some texts are being defined outside of `ResourceBundle.properties`